junos/resource_static_route_test.go

```diff
@@ -69,13 +73,17 @@ func TestAccJunosStaticRoute_basic(t *testing.T) {
 						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_ipv6_instance",
 							"next_hop.0", "st0.0"),
 						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_ipv6_instance",
-							"qualified_next_hop.#", "1"),
+							"qualified_next_hop.#", "2"),
 						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_ipv6_instance",
 							"qualified_next_hop.0.next_hop", "st0.0"),
 						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_ipv6_instance",
 							"qualified_next_hop.0.preference", "101"),
 						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_ipv6_instance",
 							"qualified_next_hop.0.metric", "101"),
+						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_ipv6_instance",
+							"qualified_next_hop.1.next_hop", "2001:db8:85a4::1"),
+						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_ipv6_instance",
+							"qualified_next_hop.1.preference", "st0.0"),
```

Comment:
it should be `qualified_next_hop.1.interface`

Reply:
Was trying to fix too late last night! :wink:

Thanks for the help!